superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -32,13 +36,33 @@ import FilterableTable from '../../components/FilterableTable/FilterableTable';
 import QueryStateLabel from './QueryStateLabel';
 import CopyToClipboard from '../../components/CopyToClipboard';
 import { prepareCopyToClipboardTabularData } from '../../utils/common';
+import { exploreChart } from '../../explore/exploreUtils';
 import { CtasEnum } from '../actions/sqlLab';
 import { Query } from '../types';
 
 const SEARCH_HEIGHT = 46;
 
+const SAVE_NEW_DATASET_RADIO_STATE = 1;
+const OVERWRITE_DATASET_RADIO_STATE = 2;
```

Comment:
Nit: it might be better to make this into an enum, since the constants are related:

```javascript
enum DatasetRadioState {
  SAVE_NEW = 1,
  OVERWRITE_DATASET = 2,
};
```